src/client/containers/ConfigurationUpdateContainer/hooks/useConfigurationDelete.ts

```diff
@@ -12,7 +14,10 @@ export const useConfigurationDelete = ({ configurationId }: UseConfigurationDele
     const [deleteConfiguration] = useMutation(requester.deleteConfiguration, { throwOnError: true });
 
     useEffect(() => {
-        if (configurationId) {
+        // This is a workaround to register remove handler on first mount only
```

Comment:
Why is this happening for `useConfigurationDelete` hook only? 

The error tells that this function (`registerOnRemoveHandler`) is called within a wrong context, it should be `remove` but it's `setting`.

This only happens if you logout then login though. So, it would be good to investigate this in details to understand why it's happening otherwise we might be suppressing a symptom of a deeper problem.